#!/usr/bin/python3


def raise_exception_msg(message=""):
    """
    Raises NameError with the error message
    'message'
    """
    raise NameError(message)
